redaxo/src/core/tests/path_test.php

```diff
@@ -45,6 +45,16 @@ public function testBasename()
         static::assertSame('config.yml', rex_path::basename('..\redaxo\data\core\config.yml'));
     }
 
+    public function testFindBinaryPath(): void
+    {
+        static::assertSame('/usr/bin/php', rex_path::findBinaryPath('php'));
```

Comment:
Ich führe PHPUnit auch gerne mal lokal aus, und da bekomme ich jetzt hier einen Fehler, weil es bei mir halt ein anderer Pfad ist.

Hatte als Idee, ob wir vielleicht mit `PHP_BINARY` vergleichen. Aber da ist das Problem, dass es den tatsächlichen Pfad bei mir enthält, während aus findBinaryPath ein homebrew-Symlink rauskommt.

Man könnte es so machen, aber keine Ahnung, ob das so sinnvoll ist als Test:

```suggestion
        static::assertSame(PHP_BINARY, realpath(rex_path::findBinaryPath('php')));
```

Reply:
Wenn es hält ok, wenn nicht kann man es nochmal ändern